lib/jcsda-emc/spack-stack/stack/cmd/stack.py

```diff
@@ -0,0 +1,39 @@
+import os
+import spack.cmd
+import spack
+import logging
+import shutil
+import llnl.util.tty as tty
+import spack.util.spack_yaml as syaml
+from spack.extensions.stack.stack_env import StackEnv, stack_path
+from spack.extensions.stack.cmd.stack_cmds.create import setup_create_parser, stack_create
+
+description = "Create spack-stack environment"
+section = "spack-stack-env"
+level = "long"
+
+subcommands = [
+    'env',
+    'container'
+]
+
+
+default_env_name = 'default'
+default_env_path = os.path.join(stack_path(), 'envs')
+default_packages = os.path.join(stack_path(),
+                                'configs', 'common', 'packages.yaml')
+container_path = os.path.join(stack_path(), 'configs' 'containers')
+
+
+def setup_parser(subparser):
+    sp = subparser.add_subparsers(metavar='SUBCOMMAND', dest='stack_command')
+    create_parser = sp.add_parser('create',
+                                  help='Create spack-stack env or container.')
+    setup_create_parser(create_parser)
+
+
+# Main command that calls subcommands
+def stack(parser, args):
+    """Look for a function called environment_<name> and call it."""
```

Comment:
`environment_<name>` or `stack_<name>` - sorry, a little confusing to me.

Reply:
This comment is no longer relevant.